Sources/SwiftDocCUtilities/ArgumentParsing/Subcommands/Convert.swift

```diff
@@ -13,331 +13,578 @@ import SwiftDocC
 import Foundation
 
 extension Docc {
-    /// Converts a documentation bundle.
+    /// Converts documentation markup, assets, and symbol information into a documentation archive.
     public struct Convert: ParsableCommand {
-
         public init() {}
-
-        // MARK: - Constants
-
+        
         /// The name of the directory docc will write its build artifacts to.
         private static let buildDirectory = ".docc-build"
-
-        // MARK: - Configuration
-
+        
+        /// The file handle that should be used for emitting warnings during argument validation.
+        ///
+        /// Provided as a static variable to allow for redirecting output in unit tests.
+        static var _errorLogHandle: LogHandle = .standardError
+        
         public static var configuration = CommandConfiguration(
-            abstract: "Converts documentation from a source bundle.")
-
-        // MARK: - Command Line Options & Arguments
-
-        /// The user-provided path to a `.docc` documentation bundle.
-        @OptionGroup()
-        public var documentationBundle: DocumentationBundleOption
-
+            abstract: "Convert documentation markup, assets, and symbol information into a documentation archive.",
+            usage: """
+            docc convert [<catalog-path>] [--additional-symbol-graph-dir <symbol-graph-dir>]
+            docc convert [<catalog-path>] [--additional-symbol-graph-dir <symbol-graph-dir>] [--output-dir <output-dir>]
+            docc convert [<catalog-path>] [--additional-symbol-graph-dir <symbol-graph-dir>] [--output-dir <output-dir>] [<availability-options>] [<diagnostic-options>] [<source-repository-options>] [<hosting-options>] [<info-plist-fallbacks>] [<feature-flags>] [<other-options>]
```

Comment:
Minor: Change `<catalog-path>`to `<docc-catalog-path>` to make it match to the INPUTS & OUTPUTS argument name